### PR TITLE
Adding icons to basic strategies section on styleguide

### PR DESCRIPTION
**Pull request by KNAthari**
### Changes
Give a short summary of the changes made in this PR. 
Added Icons to pills section of stylesguide

### Link to Issue 
(https://app.asana.com/0/1206510424537042/1206973606458186/f)

### Comments, Notes, or  Questions
I added icons from the figma wireframe to the 'support' and 'hold back' pills in the styleguide.

### Related Screenshots 
For FE tickets specifcially, please include a screenshot of your work!
Screenshot 2024-05-01 at 12 21 55 PM

### Before you submit for review, Did you...

- [x] Added relevant tests (if applicable)
- [x] Linked Asana ticket to PR
- [x] Added comments, notes, or questions for reviewwer
- [x] Added relevant screenshots
- [x] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

File: app/views/styleguides/index.html.erb [hunk 1]
```diff
@@ -83,23 +83,53 @@
       <h2 class="font-bold">Pills</h2>
       <h4>This is the Pills section of the styleguide.
       NOTE: Notice the use of font-secondary for these pills.</h4>
-
-
       <!--TO DO: Add icons for support/neutral/hold back-->
       <h3 class="mb-4 mt-5"><u>Basic Strategy</u></h3>
       <div class="flex flex-row">
-        <span class="pill support font-secondary">SUPPORT</span>
+        <span class="pill support font-secondary"> <%= image_tag 'basic_strategy_icons/hand-holding-light 2.png', class: 'float-left pr-2' %> SUPPORT </span>
```
Comment (camckin10) [inline app/views/styleguides/index.html.erb:89]: @KNAthari The icons look great, but you will need to resize the icons in the pills so the text and icon are centered in the pill.

Comment (KNAthari) [reply app/views/styleguides/index.html.erb:89]: I will do that.

Comment (KNAthari) [reply app/views/styleguides/index.html.erb]: It has been done.

Comment (camckin10) [reply app/views/styleguides/index.html.erb]: @KNAthari Nice. Update your PR description with the latest screenshot of your changes, and an apprentice approval.

**Review by camckin10 — APPROVED** [review]
Nice work! 👍

**Review by crestu — APPROVED** [review]
looks good! nice job!

**Pull request merged**